apps/projects/app/components/Shared/BountyContextMenu.js

```diff
@@ -42,6 +42,10 @@ BountyContextMenu.propTypes = {
   onReviewApplication: PropTypes.func.isRequired,
   onReviewWork: PropTypes.func.isRequired,
   workStatus: PropTypes.oneOf([undefined, 'new', 'review-applicants', 'submit-work', 'review-work', 'finished']),
+  work: PropTypes.oneOf([
```

Comment:
In this case it should be required, then you probably will get a prop error since Issues and Details are not receiving the work prop (they should have proptypes as well to help detect this bug), also I don't like the solution of passing work with "undefined" value

Reply:
That's because the "work" is embedded in an Issue and passed implicitly around - in Content/Issues.js it is passed down to IssueDetail and Card via unfolding of {...issue}. It is undefined when there is no bounty or no work has been submitted yet - otherwise it is {}.
Now, I admit that having PropTypes check like "it's there or not" is not good - maybe 'work' property can be set to an empty {} right when the Issue is first mentioned? But then again it's cleaner to check if it is defined or not instead of checking if {} is empty or not? In any case there are no errors showing up on my console.

Reply:
Ok now I understand it is embedded inside the issue object, ok, yes, I agree even this solution does not seen conventional, it works so I will approve the PR.
great job 👍